src/numpyro_ext/optim.py

```diff
@@ -80,25 +84,46 @@ def run(rng_key, *args, **kwargs):
         state = svi.init(init_key, *args, **kwargs)
         for _ in range(num_steps):
             state, _ = svi.update(state, *args, **kwargs)
+        info = state.optim_state[1].state
```

Comment:
Oh, this only works if the optimizer state itself has a `.state`, which works for the `Scipy*Minimize`ers but will fail if you use, e.g., `Adam`. So you might want to do something like this instead:
```suggestion
        info = getattr(state.optim_state[1], 'state', None)
```